includes/components/class-admin.php

```diff
@@ -138,6 +138,7 @@ public function add_admin_pages() {
 		add_submenu_page( 'hp_settings', hivepress()->translator->get_string( 'settings' ) . $title, hivepress()->translator->get_string( 'settings' ), 'manage_options', 'hp_settings', [ $this, 'render_settings' ], 0 );
 		add_submenu_page( 'hp_settings', esc_html__( 'Themes', 'hivepress' ) . $title, esc_html__( 'Themes', 'hivepress' ), 'install_themes', 'hp_themes', [ $this, 'render_themes' ] );
 		add_submenu_page( 'hp_settings', esc_html__( 'Extensions', 'hivepress' ) . $title, esc_html__( 'Extensions', 'hivepress' ), 'install_plugins', 'hp_extensions', [ $this, 'render_extensions' ] );
+		add_submenu_page( 'hp_settings', esc_html__( 'Status', 'hivepress' ) . $title, esc_html__( 'Status', 'hivepress' ), 'install_plugins', 'hp_status', [ $this, 'render_status' ] );
```

Comment:
Please use the 'manage_options' capability here, e.g. in case if some user has access to settings but can't install themes and plugins